src/pgduckdb_detoast.cpp

```diff
@@ -78,20 +81,22 @@ ToastDecompressDatum(struct varlena *attr) {
 	case TOAST_LZ4_COMPRESSION_ID:
 		return Lz4DecompresDatum(attr);
 	default:
-		elog(ERROR, "invalid compression method id %d", TOAST_COMPRESS_METHOD(attr));
+		throw duckdb::InvalidInputException("(PGDuckDB/ToastDecompressDatum) Invalid compression method id %d",
+		                                    TOAST_COMPRESS_METHOD(attr));
 		return NULL; /* keep compiler quiet */
 	}
 }
 
 static struct varlena *
 ToastFetchDatum(struct varlena *attr) {
-	Relation toastrel;
+	Relation toast_rel;
```

Comment:
sensible change, but lets do that in a separate PR (or commit) to keep history understandable.